config/environments/production.rb

```diff
@@ -93,7 +93,7 @@
 
   cache_ssl = (ENV.fetch('CACHE_SSL') { 'false' }) == 'true'
   cache_namespace = "#{ENV.fetch('CLIENT')}-#{Rails.env}-hmis"
-  redis_config = Rails.application.config_for(:cache_store).merge({ expires_in: 8.hours, raise_errors: false, ssl: cache_ssl, namespace: cache_namespace })
+  redis_config = Rails.application.config_for(:cache_store).merge({ expires_in: 8.hours, race_condition_ttl: 1.minute, ssl: cache_ssl, namespace: cache_namespace })
```

Comment:
`raise_errors` not valid option anymore. Adding `race_condition_ttl` per recommendations in documentation